main.js

```diff
@@ -1,5 +1,5 @@
 function initiateBooksData() {
```

Comment:
- [ ] please as the requirements you should also refactor this function into class so it should a method's class I encourage you to do that